### PR TITLE
Add information about export grids to CSV to demos

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/DataGrid/ExcelJSOverview/description.md [hunk 1]
```diff
@@ -3,3 +3,5 @@ To enable export in the DataGrid, reference or import the <a href="https://githu
 Once the conditions above are met, use the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) method to export the DataGrid to an Excel <a href="https://github.com/exceljs/exceljs#create-a-workbook" target="_blank">workbook</a>. 
 
 Review the [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) handler to see the data export code. DataGrid is exported as is to a single <a href="https://github.com/exceljs/exceljs#add-a-worksheet" target="_blank">worksheet</a>. You can also set [allowExportSelectedData](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#allowExportSelectedData) to **true** to export only selected rows.
+
+You can also export DataGrid to CSV. To do this, implement the **exportDataGrid(options)** method in the same way as shown in the following ticket: <a href="https://supportcenter.devexpress.com/ticket/details/t920593/pivotgrid-exceljs-export-to-export-pivotgrid-into-csv-file" target="_blank">Export PivotGrid into CSV file</a>.
```
Comment (arminal) [inline JSDemos/Demos/DataGrid/ExcelJSOverview/description.md]:
```suggestion
You can export DataGrid to CSV. Implement the **exportDataGrid(options)** method following the steps outlined in the following ticket: <a href="https://supportcenter.devexpress.com/ticket/details/t920593/pivotgrid-exceljs-export-to-export-pivotgrid-into-csv-file" target="_blank">Export PivotGrid into CSV file</a>.
```

File: JSDemos/Demos/PivotGrid/ExcelJSOverview/description.md [hunk 2]
```diff
@@ -4,3 +4,5 @@ Once you've referenced/imported both files and set **export**.**enabled** to **t
 
 
 Please review the [onExporting](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onExporting) handler and its data export code to learn more. In this example, PivotGrid content is exported as is to a single <a href="https://github.com/exceljs/exceljs#add-a-worksheet" target="_blank">worksheet</a>.
+
+You can also export PivotGrid to CSV. To do this, implement the **exportPivotGrid(options)** method as shown in the following ticket: <a href="https://supportcenter.devexpress.com/ticket/details/t920593/pivotgrid-exceljs-export-to-export-pivotgrid-into-csv-file" target="_blank">Export PivotGrid into CSV file</a>.
```
Comment (arminal) [inline JSDemos/Demos/PivotGrid/ExcelJSOverview/description.md]:
```suggestion
You can also export PivotGrid to CSV. Implement the **exportPivotGrid(options)** method as shown in the following ticket: <a href="https://supportcenter.devexpress.com/ticket/details/t920593/pivotgrid-exceljs-export-to-export-pivotgrid-into-csv-file" target="_blank">Export PivotGrid into CSV file</a>.
```

File: JSDemos/Demos/DataGrid/ExcelJSOverview/description.md (hunk 1 above)
Comment (AlexJameson) [reply JSDemos/Demos/DataGrid/ExcelJSOverview/description.md]: Hello. We have a repetition here: 'following steps outlined in the following ticket'. Can we rephrase it in some way?

Comment (AlexJameson) [reply JSDemos/Demos/DataGrid/ExcelJSOverview/description.md]: There is also an important thing - we want users to implement the export**Data**Grid(options) method, and in the ticket below we show export**Pivot**Grid(options) method. They are almost the same but I would like to tell users that they should do almost the same. Also, there is only one step shown in the ticket. 

How about the following: 

 Implement the **exportDataGrid(options)** method as shown in the following ticket:

Comment (dirk-pieterse) [reply JSDemos/Demos/DataGrid/ExcelJSOverview/description.md]: You can export DataGrid to CSV.  Implement the **exportDataGrid(options)** method as shown in the following ticket: <a href="https://supportcenter.devexpress.com/ticket/details/t920593/pivotgrid-exceljs-export-to-export-pivotgrid-into-csv-file" target="_blank">Export PivotGrid into CSV file</a>.

**Review by dirk-pieterse — DISMISSED** [review]

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**